src/main/java/com/akademia/educators_back/controller/ProblemController.java

```diff
@@ -38,6 +37,7 @@ public ProblemTo findProblemById(@PathVariable Long id){
     @PostMapping("/add")
     @ResponseStatus(HttpStatus.ACCEPTED)
     void createProblem(@RequestBody ProblemTo problemTo){
+        System.out.println("213123");
         problemService.addProblemToDB(problemTo);
```

Comment:
sugerowałbym lepszą nazwę dla metody np: "save" 
inny programista czytając: problemService.save(problemTo) widzi po tej linijce wszystko co się dzieje, widomo, że zapis odbywa się na bazie danych i skoro wywołujemy porblemService oraz w parametrze przekazujemy problemTo, to dotyczy to problemu